.gitmodules

```diff
@@ -81,3 +81,7 @@
   path = parm/jcb-algorithms
   url = https://github.com/noaa-emc/jcb-algorithms
   branch = develop
+[submodule "sorc/bufr-query"]
+  path = sorc/bufr-query
+  url = https://github.com/noaa-emc/bufr-query.git
+  branch = master
```

Comment:
develop

Reply:
@CoryMartin-NOAA I'm working on rewriting the history of bufr-query so its in a funny state right now. The main branch should be the develop branch.

Reply:
Ok thanks, just let me know when I can/should update the submodule hash and the .gitmodules file.

Reply:
@rmclaren , is it now safe to switch back to `develop`?

Reply:
Yes, you should be good to do that.

Reply:
Update to `develop` at  09a2fb9